### PR TITLE
Adding braces for if block

### PR DESCRIPTION
**Pull request by trbehera**
It is recommended to add explicit braces around if / else / for / while
blocks even if the block contains a single statement.

Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

File: crypto/openssl/openssl_key_exchange_ecdh.c [hunk 1]
```diff
@@ -245,24 +250,27 @@ static bool compute_publicBECDH(ecdh_context_t *key_ex_data)
...
 		goto exit;
+	}
 	temp[0] = tmp >> 8;
 	size = 1;
 	temp[size] = (tmp & 0x00ff);
 	size += 1;
 	size += BN_bn2bin(x, &temp[size]);
 	tmp = bn_num_bytes(y);
-	if (tmp & 0xffff0000) // check size more than 2 byte size space
+	if (tmp & 0xffff0000) {// check size more than 2 byte size space
```
Comment (johneasterday) [inline crypto/openssl/openssl_key_exchange_ecdh.c:262]: nitpick - should there be a space between the brace and comment (or comment on separate line)

File: crypto/openssl/openssl_key_exchange_ecdh.c [hunk 2]
```diff
@@ -245,24 +250,27 @@ static bool compute_publicBECDH(ecdh_context_t *key_ex_data)
...
 		goto exit;
+	}
 	temp[size] = tmp >> 8;
 	size += 1;
 	temp[size] = (tmp & 0x00ff);
 	size += 1;
 	size += BN_bn2bin(y, &temp[size]);
 	tmp = bn_num_bytes(key_ex_data->_Device_random);
-	if (tmp & 0xffff0000) // check size more than 2 byte size space
+	if (tmp & 0xffff0000) {// check size more than 2 byte size space
```
Comment (johneasterday) [inline crypto/openssl/openssl_key_exchange_ecdh.c:271]: nitpick - should there be a space between the brace and comment (or comment on separate line)

File: lib/fdonet.c [hunk 3]
```diff
@@ -405,8 +422,9 @@ bool resolve_dn(const char *dn, fdo_ip_address_t **ip, uint16_t port,
 		}
 	}
 end:
-	if (ip_list) // free ip_list
+	if (ip_list) {// free ip_list
```
Comment (johneasterday) [inline lib/fdonet.c:425]: nitpick - should there be a space between the brace and comment (or comment on separate line)

File: network/network_if_mbedos.c [hunk 4]
```diff
@@ -203,8 +209,9 @@ fdo_con_handle fdo_con_connect(fdo_ip_address_t *ip_addr, uint16_t port,
  */
 int32_t fdo_con_disconnect(fdo_con_handle handle, void *ssl)
 {
-	if (ssl) // SSL disconnect
+	if (ssl) {// SSL disconnect
```
Comment (johneasterday) [inline network/network_if_mbedos.c:212]: nitpick - should there be a space between the brace and comment (or comment on separate line)

**Review by johneasterday — COMMENTED** [review]
added a few comments

File: crypto/openssl/openssl_key_exchange_ecdh.c (hunk 1 above)
Comment (trbehera) [reply crypto/openssl/openssl_key_exchange_ecdh.c]: > added a few comments

Updated in 310e2a4f

**Review by johneasterday — APPROVED** [review]

**Review by prateekcr — APPROVED** [review]

**Pull request merged**